components/builder-api/src/config.rs

```diff
@@ -53,6 +55,7 @@ impl Default for Config {
             events_enabled: false,
             non_core_builds_enabled: false,
             log_dir: env::temp_dir().to_string_lossy().into_owned(),
+            webhook_secret_token: "".to_string(),
```

Comment:
I think its ok to put the dev secret key here since its dev and thats what we are doing with the oauth keys

Reply:
also since this is GH related, it may be better to include it in the `GitHubCfg` and then you don't have to `chain.link` a `WebhookSecretToken`

Reply:
Ah, great suggestions.  I wasn't aware of how all the other GH related stuff was hooked up.  I'll do that.